emsdk

```diff
@@ -684,7 +692,7 @@ class Tool:
   def __init__(self, data):
     # Convert the dictionary representation of the tool in 'data' to members of this class for convenience.
     for key in data:
-      if isinstance(data[key], unicode):
+      if sys.version_info < (3,) and isinstance(data[key], str):
```

Comment:
Oh, that's indeed a mistake. `2to3` changed the `unicode` to `str` and I didn't change it back again even though I adjusted the line.